packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -20,7 +18,7 @@
           placement="right"
         >
           <WarningIcon
-            :color="KUI_COLOR_TEXT_WARNING"
+            :color="`var(--kui-color-text-warning, ${KUI_COLOR_TEXT_WARNING})`"
```

Comment:
Within templates don't have to use the `var`:
```suggestion
            :color="KUI_COLOR_TEXT_WARNING"
```

Reply:
Just following what @adamdehaven suggested on the previous MetricCard PR:
https://github.com/Kong/public-ui-components/pull/1520/files#diff-3bc599b221969aa411e68aa16a21553d7e436bf2964ffd586acc64dfb501f318R11

Reply:
Yes, for colors especially we should use this syntax for any components that should be customizable (e.g. for Portal)